lib/main.js

```diff
@@ -118,6 +118,12 @@ function logServerPort (address, port) {
   logger.info(logMessage);
 }
 
+function initHeapdump (args) {
+  if (args.isHeapdumpEnabled) {
+    require('heapdump');
```

Comment:
This is pretty enough to be able to send SIGUSR2. writeSnapshot is only needed for Windows